Formula/u/uffizzi.rb

```diff
@@ -1,30 +1,16 @@
 class Uffizzi < Formula
   desc "Self-serve developer platforms in minutes, not months with k8s virtual clusters"
   homepage "https://uffizzi.com"
-  url "https://github.com/UffizziCloud/uffizzi_cli/archive/refs/tags/v2.3.2.tar.gz"
-  sha256 "8437dd01ae74b4de562766700ab71cf0346bf354bbf32451d273ea90c954ad8c"
+  url "https://github.com/UffizziCloud/uffizzi_cli/archive/refs/tags/v2.3.3.tar.gz"
+  sha256 "72f60f25350803ecb71b3ca20020393f76914c0eeb1b59187d500fbfab89a1d1"
   license "Apache-2.0"
 
-  livecheck do
-    url :stable
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
-  end
-
-  bottle do
-    sha256 cellar: :any,                 arm64_sonoma:   "7775093bbe792b5c8a98159d3e09294fe5cf651e67273d8f28e24cb57bc00995"
-    sha256 cellar: :any,                 arm64_ventura:  "5b48032ccdc363005e2839144d8ddaea81f85f53947cffd0dfaf24baaa1b20bc"
-    sha256 cellar: :any,                 arm64_monterey: "595a22c97103a4131cd73df11219065a4b1a5a3f3054bc1771799e1cd5dc1322"
-    sha256 cellar: :any,                 sonoma:         "aa110074e8efe4c68559785320d19975c3fbda888cd2db0d71e15ca66893324a"
-    sha256 cellar: :any,                 ventura:        "b0ea91e2ed4b9f741461a076659d92506dd31c1fbf6858226d23e77c1311fa46"
-    sha256 cellar: :any,                 monterey:       "77d7e4c37b5f6d7cd4a2867662fcd38d199670cf718ffe4ba6615902ca688ccb"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "5b65edddec7936934aedbc201fff91e7884d7fbc93ca7b1d2c0b0eaa970fca47"
-  end
-
```

Comment:
dont remove these, thanks! :)

Reply:
Sure, I thought this is added automatically each time